webpack.config.js

```diff
@@ -120,6 +120,9 @@ const devConfig = {
   plugins: [
     // Include hot reloading functionality
     new webpack.HotModuleReplacementPlugin(),
+    new webpack.DefinePlugin({
+      'window.API_URL': process.env.API_URL ? `"${process.env.API_URL}"` : false,
+    }),
```

Comment:
Don't forget to add comments when making changes to `webpack.config.js` so that the purpose of this plugin is clear to a hypothetical person who is unfamiliar with the code.